profile/README.md

```diff
@@ -1,26 +1,32 @@
 <p align="left">
-    <a href="https://stacks.co"><img alt="Stacks" src="https://github.com/stacks-network/.github/blob/master/profile/images/stacks-logo.png" width="250"/></a>
+    <a href="https://stacks.co"><img alt="Stacks" src="https://github.com/stacks-network/.github/assets/6131188/bd7a652a-f1ed-4dc0-8acc-247f396a7eb2" width="200"/></a>
```

Comment:
what about updating the image located here - `./profile/images/stacks-logo.png` ?
otherwise, can you delete the existing image (i can help there if you need it)